tools/fuzzy_proof_generator_tool/test/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+npm run compile:ts && \
+mocha -r ts-node/register tools/fuzzy_proof_generator_tool/test/**/*.test.ts && \
```

Comment:
You don't need to compile if you use the `ts-node` register. You are basically compiling twice.

Reply:
In that case you can run mocha with the compiled JS code and don't need to add the ts-node register.

Reply:
`npm run compile:ts` only compiles the sources, and not tests. tsconfig.json was configured so. That's why it's still needed to run mocha with ts-node. Or I am missing something big here?